app/services/customers/create_service.rb

```diff
@@ -162,7 +165,7 @@ def create_billing_configuration(customer, billing_configuration = {})
       create_provider_customer ||= billing_configuration[:provider_customer_id]
       return unless create_provider_customer
 
-      customer.update!(payment_provider: billing_configuration[:payment_provider]) if api_context?
+      customer.update!(payment_provider: billing_configuration[:payment_provider], payment_provider_code: billing_configuration[:payment_provider_code]) if api_context?
```

Comment:
Style/IfUnlessModifier: Modifier form of `if` makes the line too long. (https://rubystyle.guide#if-as-a-modifier)